src/components/history-control-panel.js

```diff
@@ -89,7 +89,7 @@ function HistoryControlPanel({
         >
         {t('permalink')}
       </Button>
-      {canRestoreRevisions && (
+      {!!canRestoreRevisions && (
```

Comment:
Funny how these were not already changed with the switch to node18? 🤔 All other such cases were caught by the linter then.

Reply:
It was in the ignored list in `.eslintrc` before :-)